modules/remotecontrol/Receiver.js

```diff
@@ -284,6 +284,19 @@ export default class Receiver extends RemoteControlParticipant {
     }
 
     /**
+     * Sends remote control event of type start.
+     *
+     * @returns {void}
+     */
+    _sendStartAPIEvent() {
+        transport.sendEvent({
+            name: REMOTE_CONTROL_EVENT_NAME,
+            type: EVENT_TYPES.start,
+            id: APP.conference.getSourceId()
+        });
+    }
+
+    /**
      * Handles remote control events from the external app. Currently only
      * events with type = EVENT_TYPES.supported or EVENT_TYPES.permissions.
```

Comment:
Probably need to update this comment too because you changed supported event types.